version.php

```diff
@@ -24,8 +24,8 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version = 2023150900;
+$plugin->version = 2023122200;
```

Comment:
This is technically I downgrade, because the first number was actually wrong (month: 15 :facepalm: )
Easiest solution: wait till 2024.01.01 and create a beta.2 with the correct date